src/__tests__/logFormatting.js

```diff
@@ -64,7 +64,7 @@ describe('logFormatting', () => {
         const stackTrace = a(req);
         expect(stackTrace[0][0]).toEqual('b');
         expect(stackTrace[0][1]).toEqual('/logFormatting.js:8:26');
-        expect(stackTrace[2][0]).toEqual('a');
+        expect(stackTrace[2][0]).toEqual('Object.it');
```

Comment:
These tests are somewhat sketchy, but I'm not going to fight that right now.

Reply:
When then?

Reply:
When I or someone else decides it's a big enough problem that we need to address it. I don't want to scope creep here. I only have a few days to get this project done and I'd like to finish with enough time to document my learnings and move on.